### PR TITLE
Add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Xiaochao8**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  Please refer to the [general guideline](https://github.com/vmware/singleton/blob/master/CONTRIBUTING.md).
+  Signature is mandatory for the commits, please make sure to get it worked. If not, your PR won't be able to merge.
+
+## Develop, Build and Test
+### Create your branch
+Changes should be made on your own fork in a new branch. The branch should be named g11n-java-client-XXX-description where XXX is the number of the issue. Follow below steps to create a new branch.
+* Add vmware/singleton as a remote repository
+
+  ```
+  git remote add vmware https://github.com/vmware/singleton.git
```
Comment (linr211) [inline CONTRIBUTING.md]: 'vmware' branch looks like not appropriate though you can add a random name, please use other more readable name.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    ``` 
+    git clone --branch g11n-java-client https://github.com/{github_user_id}/singleton.git g11n-java-client
+    ```
+
+* Set Signature, setup CI Pipeline 
+  Please refer to the [general guideline](https://github.com/vmware/singleton/blob/master/CONTRIBUTING.md).
+  Signature is mandatory for the commits, please make sure to get it worked. If not, your PR won't be able to merge.
+
+## Develop, Build and Test
+### Create your branch
```
Comment (tigershi) [inline CONTRIBUTING.md]: The same format as Fork repository is not used

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
+# Contributing to Singleton Java client
+
+
+## Welcome
+
+Welcome to Singleton Java client! This guide provides information on filing issues and guidelines for open source contributors. **Please leave comments / suggestions if you find something is missing or incorrect.**
+
+## Get Started
```
Comment (linr211) [inline CONTRIBUTING.md]: Looks like there are some overlaps with this doc: https://github.com/vmware/singleton/blob/master/CONTRIBUTING.md
so, I would suggest separate the common part and give above link for it, and this guideline only includes Java client specific.

Comment (Xiaochao8) [reply CONTRIBUTING.md]: The doc has already been refered to.

**Pull request merged**